privacyidea/lib/utils.py

```diff
@@ -47,24 +45,22 @@
 import hashlib
 import traceback
 import os
+from random import SystemRandom
 
-try:
-    import bcrypt
-    _bcrypt_hashpw = bcrypt.hashpw
-except ImportError:  # pragma: no cover
-    _bcrypt_hashpw = None
+from privacyidea.lib.error import ParameterError, ResourceNotFoundError
 
 # On App Engine, this function is not available.
 if hasattr(os, 'getpid'):
     _pid = os.getpid()
 else:  # pragma: no cover
     # Fake PID
-    _pid = urandom.randint(0, 100000)
+    _pid = SystemRandom().randint(0, 100000)
```

Comment:
Hm, do we even use the ``_pid`` variable? I don't see any usage :-/ If we don't use it, I guess we could just delete the snippet above.

Reply:
You are right, this was used to initialize a pseudo rng which i removed in #1374 
I'll fix this.

Reply:
Fixed in 22a489a

Reply:
:+1:  thanks!